.eslintrc

```diff
@@ -22,7 +22,8 @@
   "rules": {
     "strict": ["error", "never"],
     "react-hooks/rules-of-hooks": "error",
-    "react-hooks/exhaustive-deps": "off"
+    "react-hooks/exhaustive-deps": "off",
+    "react/prop-types": "off"
```

Comment:
I moved this rules to be disabled everywhere but not sure if it was the best way to go, feedback welcomed
Other options could have been:
- Add prop-types, we would need to add the library tho as it wasn't installed
- Use TS in stories, I don't have context on why we are using JS in story

Reply:
No idea why we use js in stories. My preference would be to migrate them in Typescript if possible

Reply:
Agree, it would be better if they were in TS

Make it a warning might be the better option for now 👍